releasenotes/notes/context-df929fc8dfe148ba.yaml

```diff
@@ -0,0 +1,15 @@
+---
+prelude: >
+    Several deprecations have been made to `Context` as we prepare to move
```

Comment:
~~~
/home/circleci/project/docs/release_notes.rst:14: Spell check: deprecations: Several deprecations have been made to .
~~~
🙁